src/common.js

```diff
@@ -284,30 +284,26 @@ function wrap(cmd, fn, options) {
         //    `cp([file1, file2, file3], dest);`
         // equivalent to:
         //    `cp(file1, file2, file3, dest);`
-        args = args.reduce(function(accum, cur) {
-          if (Array.isArray(cur)) {
+        args = args.reduce(function (accum, cur) {
+          if (Array.isArray(cur))
             return accum.concat(cur);
-          } else {
-            accum.push(cur);
-            return accum;
-          }
+          accum.push(cur);
+          return accum;
         }, []);
 
         // Convert ShellStrings (basically just String objects) to regular strings
-        args = args.map(function(arg) {
-          if (arg instanceof Object && arg.constructor.name === 'String') {
+        args = args.map(function (arg) {
+          if (arg instanceof Object && arg.constructor.name === 'String')
             return arg.toString();
-          } else
-            return arg;
+          return arg;
         });
 
         // Expand the '~' if appropriate
         var homeDir = getUserHome();
-        args = args.map(function(arg) {
+        args = args.map(function (arg) {
           if (typeof arg === 'string' && arg.slice(0, 2) === '~/' || arg === '~')
```

Comment:
Nit: Can we either add curly braces here, or have the `return` on the same line as the `if`?